src/tike/ptycho/solvers/divided.py

```diff
@@ -8,9 +8,10 @@
 
 
 def divided(
-    self,
+    op,
     data, probe, scan, psi,
-    recover_psi=True, recover_probe=False, recover_positions=False,
+    recover_psi=True, recover_probe=True, recover_positions=False,
```

Comment:
do you mean save_psi or recover_psi? Because we always recover_psi no matter which method is used. Maybe use 'update' instead of 'recover' for those?

Reply:
No, we don't always recover psi. When testing whether a new subproblem solver works, we hold all the other subproblems constant. Update is probably a better word though.

Reply:
I can't make this change because then functions which do the updates have the same name as the parameter. Unless I prefix the function names with underscores? Then the code would look something like this:

```python
if update_positions:
        scan, cost = _update_positions(op, nearplane, psi, probe, scan)
```